README.md

```diff
@@ -1 +1,180 @@
-# capstoneProject2
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📗 Table of Contents](#-table-of-contents-1)
+- [📖 \[to-do-list\] ](#-to-do-list-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+    - [Deployment](#deployment-1)
+  - [👥 Authors ](#-authors--1)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [Live Link:](#live-link)
+  - [Show your support](#show-your-support)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 [Capstone-Project 2] <a name="Capstone-Project"></a>
+
+**[Capstone-Project 2]** is a program that fetch data from an API and shows them in a webpage and allows the user to interact (like, comment, etc.) trhough another API
```

Comment:
- Optional. Consider giving your project a proper name. Something like **MealsDB**, along this line. But it is up to you.